tests/resources/importer/HHComplete/PotassiumChannelModel.cellml

```diff
@@ -0,0 +1,186 @@
+    <variable name="V" units="mV" interface="public_and_private"/>
+    <variable name="t" units="ms" interface="public_and_private"/>
+    <variable name="n" units="dimensionless" interface="private"/>
+    <math xmlns="http://www.w3.org/1998/Math/MathML" xmlns:cellml="http://www.cellml.org/cellml/2.0#">
+      <apply>
+        <eq/>
+        <ci>i_K</ci>
+        <apply>
+          <times/>
+          <apply>
+            <power/>
+            <ci>n</ci>
+            <cn cellml:units="dimensionless">4</cn>
+          </apply>
+          <ci>g_K</ci>
+          <apply>
+            <minus/>
+            <ci>V</ci>
+            <ci>E_K</ci>
+          </apply>
+        </apply>
+      </apply>
+    </math>
+  </component>
+  <component name="nGate">
+    <variable name="n" units="dimensionless" interface="public_and_private"/>
+    <variable name="t" units="ms" interface="public_and_private"/>
+    <variable name="V" units="mV" interface="public_and_private"/>
+  </component>
+  <component name="nGateEquations">
+    <variable name="t" units="ms" interface="public_and_private"/>
+    <variable name="V" units="mV" interface="public"/>
+    <variable name="alpha_n" units="per_ms" interface="private"/>
+    <variable name="beta_n" units="per_ms" interface="private"/>
+    <variable name="n" units="dimensionless" interface="public_and_private"/>
+    <math xmlns="http://www.w3.org/1998/Math/MathML" xmlns:cellml="http://www.cellml.org/cellml/2.0#">
+      <apply>
+        <eq/>
+        <ci>alpha_n</ci>
+        <apply>
+          <divide/>
+          <apply>
+            <times/>
+            <cn cellml:units="per_mV_ms">-0.01</cn>
+            <apply>
+              <plus/>
+              <ci>V</ci>
+              <cn cellml:units="mV">65</cn>
+            </apply>
+          </apply>
+          <apply>
+            <minus/>
+            <apply>
+              <exp/>
+              <apply>
+                <divide/>
+                <apply>
+                  <plus/>
+                  <ci>V</ci>
+                  <cn cellml:units="mV">65</cn>
+                </apply>
+                <cn cellml:units="mV">-10</cn>
+              </apply>
+            </apply>
+            <cn cellml:units="dimensionless">1</cn>
+          </apply>
+        </apply>
+      </apply>
+      <apply>
+        <eq/>
+        <ci>beta_n</ci>
+        <apply>
+          <times/>
+          <cn cellml:units="per_ms">0.125</cn>
+          <apply>
+            <exp/>
+            <apply>
+              <divide/>
+              <apply>
+                <plus/>
+                <ci>V</ci>
+                <cn cellml:units="mV">75</cn>
+              </apply>
+              <cn cellml:units="mV">-80</cn>
+            </apply>
+          </apply>
+        </apply>
+      </apply>
+    </math>
+  </component>
+  <component name="nGateParameters">
+    <variable name="n" units="dimensionless" initial_value="0.325" interface="public"/>
```

Comment:
interesting to see how the analyser would handle this model... As far as I can tell, `n` is connected to `X` in the imported model, which in turn has its own initial value set... (doesn't affect this PR, just a general interest question!)